README.md

```diff
@@ -188,9 +188,19 @@ It's also possible to customize the url by appending _query parameters_. Just ap
 
 Want to use custom templates and keep them version controlled in the same repository? Use `file:` schema to load templates directly from the file system! The below code will load the template from the path `./templates/vanilla-console`, relative to the markdown file. The file templates are directories with at least a `package.json` file inside.
 
+As in the other examples, the content of the code block will replace the entry file in the template.
+
 ````md
 ```js codesandbox=file:./templates/vanilla-console
-// ...
+console.log('this code will replace the entry file content');
+```
+````
+
+If you would like to use the template as-is without any of the content of the code block, add the `?overrideEntry=false` query string:
```

Comment:
A disclaimer of this query param will NOT be passed to the generated CodeSandbox would be nice 👍 .

Reply:
good idea, done 👍